beancount_ethereum/importer.py

```diff
@@ -37,17 +37,36 @@ def account_map(self):
         return {key.lower(): value for key, value
                 in self.config['account_map'].items()}
 
+    def account_suffix(self, currency):
+        if 'currency_map' in self.config:
+            for keyval in self.config['currency_map']:
+                if currency.lower() == keyval['blockchain_currency'].lower():
+                    return keyval['account_suffix']
+            return currency
+        else:            
+            return currency
```

Comment:
I believe in most cases account suffix will be the same as commodity name so we can make `account_suffix` optional and use commodity name if suffix is not defined.

Reply:
Sure.